hoot-core-test/src/test/cpp/hoot/core/test/ConflateCaseTestSuite.cpp

```diff
@@ -22,16 +22,16 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015, 2016, 2017 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2015, 2016, 2017, 2018 DigitalGlobe (http://www.digitalglobe.com/)
  */
 #include "ConflateCaseTestSuite.h"
 
 // hoot
-#include <hoot/core/util/HootException.h>
-#include <hoot/core/util/Log.h>
 #include <hoot/core/HootConfig.h>
 #include <hoot/core/cmd/ConflateCmd.h>
-#include "ConflateCaseTest.h"
+#include <hoot/core/test/ConflateCaseTest.h>
```

Comment:
All `#include`s followed by double quotes except for the corresponding `.h` for this `.cpp` where converted to paths within the dependency structure.